### ISSUE TITLE
Trigger search button functionality via enter key

### ISSUE THREAD

**Issue by resant18** [gwg-women/gwg-women-techmakers#169]
Add functionality that allows user to use the enter key to submit the keyword for searching place.

**Comment by digilou** [comment]
Oh no! I thought this was working. The `input type="search"` does work with the `Enter key`, but the button doesn't?

**Comment by digilou** [comment]
Sorry, I just now tested this. The hitting `Enter` for the search button works. But hitting `Enter` after autocomplete within the box does NOT work for me.

**Comment by resant18** [comment]
Yeah, it doesn't work when user type their own keyword and not choosing from the autocomplete list.

**Comment by digilou** [comment]
I'll take a stab at this, since no one else has grabbed it yet.

**Comment by khusbuchandra** [comment]
I went off the radar for a couple of day but I am available now. @digilou , let me know if you need help on this.

**Comment by digilou** [comment]
@khusbuchandra Please feel free to take a look! I haven't started on it yet.

**Comment by khusbuchandra** [comment]
As I work in completely different time zone now, if you see it's still not taken by your morning . Feel free to pick. Currently pairing up with Angie and Sonali for offline capabilities. If that gets sorted will try my hand here.
